src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReplacementServiceTests.cs

```diff
@@ -23,25 +23,142 @@
 using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationErrorMessages;
 using static GovUk.Education.ExploreEducationStatistics.Common.Model.TimeIdentifier;
 using FootnoteService = GovUk.Education.ExploreEducationStatistics.Data.Model.Services.FootnoteService;
+using IReleaseService = GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.IReleaseService;
 using Release = GovUk.Education.ExploreEducationStatistics.Data.Model.Release;
+using Unit = GovUk.Education.ExploreEducationStatistics.Common.Model.Unit;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
 {
     public class ReplacementServiceTests
     {
         private const string CountryCodeEngland = "E92000001";
 
+        [Fact]
+        public async Task GetReplacementPlan_ReleaseFileReferenceHasWrongFileType()
+        {
+            var originalSubject = new Subject
+            {
+                Id = Guid.NewGuid(),
```

Comment:
Dunno if we need to be going to the trouble of assigning ids to everything (more lines than we need)? I've found ids get generated automatically when the entities get persisted into the context.

Same question applies to the rest of this file.

Reply:
Thanks, I've remove a ton of lines now where new Id's are initialised except where they are needed by references before saving like in the ReleaseFileReference.SubjectId field and in the DataBlock  Filters/Indicators fields. 

I tried reworking the tests to save these first before saving other stuff like the DataBlocks but the layout then just became a mess.

Reply:
Nice one :+1: